docs/_sass/custom/custom.scss

```diff
@@ -63,7 +63,6 @@ body {
 
 .main {
   min-height: 100vh;
-  display: flex;
```

Comment:
🙈 that this works (I've confirmed that), but I'm mystified why it does.

At some point I think we should look at the background color applied to that area, as the blue-on-gray I don't think passes a contrast check:

![Screenshot 2023-08-21 at 1 42 16 PM](https://github.com/MITLibraries/timdex/assets/1403248/3d2919d3-ebe6-4459-9d74-f651847ef6a4)

(WebAIM has this at a 1.69:1 ratio right now, assuming this change does what I think it will. I'm fine with doing this in a subsequent PR, though - the contrast problem exists now, so we aren't introducing it here.)

Reply:
I've increased contrast for this search preview in both light and dark modes. It likely isn't perfect yet but it is better. I am having a bit of a hard time actually getting the contrast report to display for the hidden content so I could use a pairing session at some point to better understand how to do that.